conf/juicer-config.yaml.template

```diff
@@ -0,0 +1,3 @@
+juicer:
+   servers:
+      redis_url: redis://127.0.0.1:6379
```

Comment:
It would be nice to include additional configurations expected by the server here.
Then the user would be able to create its own config file based on this template.

We can update this pull request or save for later.